crates/web5/src/credentials/presentation_definition.rs

```diff
@@ -172,11 +173,10 @@ impl InputDescriptor {
         for vc_jwt in vc_jwts {
             let mut selection_candidate: Map<String, Value> = Map::new();
 
-            let decoded_jws = CompactJws::decode(vc_jwt)?;
-            let payload_json = String::from_utf8(decoded_jws.payload).map_err(|_| {
-                PexError::JsonError(
-                    "Could not create json string from vc jwt payload bytes".to_string(),
-                )
+            let vc = VerifiableCredential::verify(vc_jwt)
```

Comment:
this is what's making the test vectors fail now, because `did:key` isn't supported here, but the test vectors use VCs created with `did:key`'s

Reply:
Actually we should slightly adjust this to use a continue statement instead of failing. The algorithm here should account for failed verification among the set of VCs and still be passable, because the intent of the function isn't to verifying everything but to select matching credentials, and for the given matches we should verify.